### PR TITLE
Feature/lockup logo

### PR DESCRIPTION
**Pull request by ScottAntonacAP**
## Summary of Changes

- Add the logo lockup as a custom view
- Add the logo lockup as an option for the price breakdown

Screen Shot 2022-04-07 at 12 51 52

## Submission Checklist

- [x] Documentation is changed or added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,6 +340,20 @@ priceBreakdownView.showInterestFreeText = false
 
 Given the above, the price breakdown text will be rendered `or 4 payments of $##.## with`
 
+### Logo Type
+Setting `logoType` is optional, will default to `.badge` and must be of type `LogoType`.
+
+Can be either of `.badge` of `.lockup`.
```
Comment (benfterpay) [inline README.md:346]:
```suggestion
Can be either of `.badge` or `.lockup`.
```

File: README.md [hunk 2]
```diff
@@ -329,6 +340,20 @@ priceBreakdownView.showInterestFreeText = false
...
+When setting color scheme on logo type of `.lockup`, only the foreground color will be applied. (See example)
+
+```swift
+let priceBreakdownView = PriceBreakdownView()
+priceBreakdownView.logoType = .lockup
+priceBreakdownView.colorScheme = .dynamic(lightPalette: .blackOnWhite, darkPalette: .whiteOnBlack)
+```
+
+Given the above, the price breakdown will contain the lockup logo and will be be colored black when the theme is white and white when the theme is dark.
```
Comment (benfterpay) [inline README.md:355]:
```suggestion
Given the above, the price breakdown will contain the lockup logo and will be colored black when the theme is white and white when the theme is dark.
```

File: Sources/Afterpay/Views/LogoView.swift [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    notificationCenter.addObserver(self, selector: selector, name: name, object: nil)
+  }
+
+  override public func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    super.traitCollectionDidChange(previousTraitCollection)
+    updateImage(withTraits: traitCollection)
+  }
+
+  internal func getImageName(brand: String, color: String) -> String {
+    print("is getting: LogoView")
```
Comment (benfterpay) [inline Sources/Afterpay/Views/LogoView.swift:53]: Is this print for debugging? If so, should it be removed before this is merged?

**Review by benfterpay — CHANGES_REQUESTED** [review]
Can the sample PNG [Images/lockup-mint.png](https://github.com/afterpay/sdk-ios/pull/211/files#diff-73929590bf312847532de8d361450323f997998ef616ce9b6e29ba1f4b9eb325) please be amended to use a dark background?

File: Sources/Afterpay/Views/LogoView.swift (hunk 3 above)
Comment (ScottAntonacAP) [reply Sources/Afterpay/Views/LogoView.swift]: Addressed in c990f90

**Comment by ScottAntonacAP** [comment]
> Can the sample PNG [Images/lockup-mint.png](https://github.com/afterpay/sdk-ios/pull/211/files#diff-73929590bf312847532de8d361450323f997998ef616ce9b6e29ba1f4b9eb325) please be amended to use a dark background?

Addressed in 1c1ea9e

**Review by benfterpay — APPROVED** [review]

**Pull request merged**